test/WebJobs.Script.Tests.Integration/EndToEndTestFixture.cs

```diff
@@ -77,6 +78,8 @@ protected EndToEndTestFixture(string rootPath, string testId)
 
         public Microsoft.ServiceBus.Messaging.QueueClient ServiceBusQueueClient { get; private set; }
 
+        public Documents.Client.DocumentClient DocumentClient { get; private set; }
```

Comment:
Can you add usings for `Microsoft.Documents`/`Microsoft.Documents.Client` so this (and some other lines) is a little shorter?